sys/dev/efidev/efirt.c

```diff
@@ -548,11 +550,12 @@ efi_call(struct efirt_callinfo *ecp)
     ((uintptr_t)cheri_sealentry(cheri_andperm(			\
 	cheri_setaddress(kernel_root_cap,			\
 	    ((struct efi_rt *)efi_phys_to_kva((uintptr_t)	\
-	    efi_runtime))->method), CHERI_PERMS_KERNEL_CODE)))
+		efi_runtime, sizeof(struct efi_rt)))->method),	\
+	    CHERI_PERMS_KERNEL_CODE)))
 #else
 #define	EFI_RT_METHOD_PA(method)				\
     ((uintptr_t)((struct efi_rt *)efi_phys_to_kva((uintptr_t)	\
-    efi_runtime))->method)
+	efi_runtime, sizeof(struct efi_rt)))->method)
```

Comment:
The old indentation here is what style(9) wants FWIW.

Reply:
Fixed, I think emacs played a trick on me.